packages/eyes/src/eyes-test/eyes.ts

```diff
@@ -0,0 +1,142 @@
+
+  return {
+    appName: projectName,
+    branchName,
+    batchName: projectName,
+    browser: {
+      name: 'chrome',
+      width: 400,
+      height: 300
+    }
+  };
+}
+
+function getStaticResources(cssFilenames: string[], resourceDir: string) {
+  const resources: any = {};
+  for (const cssFilename of cssFilenames) {
+    const url = '/' + cssFilename;
+    resources[url] = {
+      url,
+      type: 'text/css',
+      value: fs.readFileSync(path.join(resourceDir, cssFilename))
+    };
+  }
+  return resources;
+}
+
+function getStatusIcon(result: any) {
+  return (
+    result.status === 'unmodified' ? chalk.green('=') :
+    result.status === 'new' ? chalk.green('+') :
+    result.status === 'modified' ? chalk.red('≠') :
+    chalk.red('!')
+  );
+}
+
+// We cannot allow any of the result promises to reject, because we're going to
+// await on a bunch of them in parallel using Promise.all, and a single rejected
+// promise would reject the entire batch.
+function getTestResult(testResultPromise: any) {
+  return (
+    testResultPromise
+    .catch((err: any) => err)
+    .then((res: any) => res instanceof TestFailedError ? res.getTestResults() : res)
+    .then((res: any) => Array.isArray(res) && res[0] instanceof TestResults ? res[0] : res)
+    .then((res: any) => {
+      if (res instanceof TestResults) {
+        if (res.getIsDifferent()) {
+          return {status: 'modified', url: res.getUrl()};
+        }
+        if (res.getIsNew()) {
+          return {status: 'new'};
+        }
+        if (res.isPassed()) {
+          return {status: 'unmodified'};
+        }
+      }
+
+      return {status: 'error', error: res};
+    })
+  );
+}
+
+async function runTest(gridClient: any, gridClientConfig: any, testName: any, html: any, resources: any) {
+  const dom = new JSDOM(html).window.document;
+
+  const {checkWindow, close} = await gridClient.openEyes({
+    ...gridClientConfig,
+    testName,
+  });
+
+  await checkWindow({
+    url: 'http://localhost/html/index.html',
+    cdt: domNodesToCdt(dom),
+    sizeMode: 'viewport',
+    resourceContents: resources
+  });
+
+  return close();
+}
+
+export async function runEyes() {
+  const resourceDir   = path.join(process.cwd(), 'dist');
+  const cssFilenames  = glob.sync('*.css', {cwd: resourceDir});
+  const htmlFilenames = glob.sync('snapshots/*.snapshot.html', {cwd: resourceDir});
+
+  const config = getGridClientConfig();
+  const resources = getStaticResources(cssFilenames, resourceDir);
+  const gridClient = makeVisualGridClient(initConfig());
+
+  const testCount = htmlFilenames.length;
+  let resultCount = 0;
+  const resultPromises = [];
+
+  for (const htmlFilename of htmlFilenames) {
+    // tslint:disable-next-line:no-console
+    console.log(`Submitting ${htmlFilename}`);
+    const html = fs.readFileSync(path.join(resourceDir, htmlFilename), 'utf-8');
+
+    const result = getTestResult(runTest(
+      gridClient,
+      config,
+      htmlFilename,
```

Comment:
What is the format of the filename, does it contain the component name, style name, and simulation name?